screens/ProfileScreen.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import {
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  View,
+  TextInput,
+  TouchableOpacity,
+  Image,
+  Button,
+} from "react-native";
+import { connect } from "react-redux";
+import firebase from 'firebase';
+
+const ProfileScreen = (props) => {
+    const [heightFeet, setHeightFeet] = useState('');
+    const [heightInch, setHeightInch] = useState('');
+    const [weight, setWeight] = useState('');
+
+  const updateUserInfo = () => {
+    firebase.database().ref('users/' + props.displayName).set({
+      weight: weight,
```

Comment:
What'd happens, if a user forgot to input weight for example. Will the request still go through?